blockchain/chain/src/store/chain_store.rs

```diff
@@ -534,6 +529,61 @@ where
         Ok(digest)
     }
 
+    /// Walks over tipset and state data and loads all blocks not yet seen.
+    /// This is tracked based on the callback function loading blocks.
+    async fn walk_snapshot<F, T>(
```

Comment:
If you move this function down a few lines then the diff will be prettier.

Reply:
Done